docs/arch.md

```diff
@@ -283,4 +283,11 @@ health_statuses_insights{metric="important"} 0
 health_statuses_insights{metric="low"} 1
 health_statuses_insights{metric="moderate"} 1
 health_statuses_insights{metric="total"} 2
-```
\ No newline at end of file
+```
+
+## Configuring what to gather
+In the yaml config there is a field named `gather` it expects a list of strings, each string is an id that is connected to a gather function.
+Adding such an id to the list means that that certain gather function needs to be run.
```

Comment:
What about to add some example of the list here?

Reply:
good idea